packages/dds/matrix/package.json

```diff
@@ -87,6 +87,7 @@
 		"pack:tests": "tar -cf ./matrix.test-files.tar ./src/test ./dist/test ./lib/test",
 		"test": "npm run test:mocha",
 		"test:coverage": "c8 npm test",
+		"test:execution-time-profiling:report": "mocha --config src/test/time/.mocharc.cjs",
```

Comment:
Let's rename the script to `test:benchmark:report`. Our infrastructure that runs perf tests expects that script name for any package that has execution time tests, so they run automatically in our pipelines.

Also, let's add this package to [this list](https://github.com/microsoft/FluidFramework/blob/3ab16ed6b84b2aec0587c5a3ee761435f153ad0f/tools/pipelines/test-perf-benchmarks.yml#L34-L40), where we define the packages where our pipelines try to run perf tests.

Reply:
Added and updated, thx!